models/agent_helpers.go

```diff
@@ -339,6 +339,8 @@ type CreateAgentParams struct {
 	TLSSkipVerify         bool
 	QueryExamplesDisabled bool
 	MaxQueryLogSize       int64
+	AWSAccessKeyID        string
+	AWSSecretAccessKey    string
```

Comment:
Let's call them 'AWS Access Key` and `AWS Secret Key` everywhere.

Reply:
Those are the names in the AWS Go SDK.
I'll change them but just to explain why I chose those names: 
https://github.com/aws/aws-sdk-go/blob/v1.25.19/aws/credentials/static_provider.go#L25-L26

Reply:
No deep reason, they are just inconsistent across AWS SDKs. For example, that linked function accepts `id` (suffix of the name) and `secret` (prefix of the name). Go SDK also accepts `AWS_ACCESS_KEY_ID`/`AWS_ACCESS_KEY` and `AWS_SECRET_ACCESS_KEY`/`AWS_SECRET_KEY` environment variables.

rds_exporter uses AWS Access Key and AWS Secret Key: https://github.com/percona/rds_exporter/blob/6d645004a7ad31e8fa43c92f2acde59e5be6703a/config/config.go#L24-L25

I don't mind much, really, as long as names are consistent in pmm-managed at least (and preferably across PMM).